### ISSUE TITLE
Reduce website loading jank

### ISSUE THREAD

**Issue by vikaspotluri123** [swapagarwal/swag-for-dev#366]
With #343, we updated the default sort order of the swag - from alphabetical to newest first. We didn't update the order that the swag is listed at build time. This leads to jank:

![jank](https://user-images.githubusercontent.com/4383564/66220890-7f30db00-e693-11e9-8904-3e4404be1e30.gif)

The solution to this is to update the `get-data` script we use to return the swag list, sorted by date.

https://github.com/swapagarwal/swag-for-dev/blob/9e4df0b188e460f26df81cfb7a32e2fa5ab2f613/get-data.js#L3-L26

**Comment by r4zorgeek** [comment]
@vikaspotluri123 Hey I am new to open source. Can I take this issue?

**Comment by vikaspotluri123** [comment]
@r4zorgeek someone has already opened a PR for this. If they don't respond in the next day or so, I'll let you know that you can work on it

**Comment by r4zorgeek** [comment]
@vikaspotluri123 Thank You.

**Comment by vikaspotluri123** [comment]
@r4zorgeek I've assigned you

**Comment by r4zorgeek** [comment]
@vikaspotluri123 When I am running gulp command this error is coming. Can you please help me with this?
![Screenshot 2019-10-14 at 10 15 40 PM](https://user-images.githubusercontent.com/14335686/66768519-5f3dba80-eed0-11e9-9952-8947148ea067.png)

**Comment by vikaspotluri123** [comment]
It looks like something failed to download. If you run it again, it will probably work

**Comment by r4zorgeek** [comment]
@vikaspotluri123 I have tried it multiple times still same error.

**Comment by r4zorgeek** [comment]
@vikaspotluri123 Hey can you please help the error is still on.

**Comment by vikaspotluri123** [comment]
Are you working on the latest version of mater?

**Comment by r4zorgeek** [comment]
@vikaspotluri123 yes sir, still the same issue.

**Comment by sanjpareek** [comment]
@r4zorgeek I was facing the same issue. There was one dependency which was not being installed with npm install and I had to do it manually - acorn@^6.0.0. Here are the steps I followed which fixed it for me:

npm cache clean -f
npm install acorn@^6.0.0
npm install
gulp

**Comment by r4zorgeek** [comment]
@sanjpareek Thanks for your comment. I tried those commands but still not fixed.

**Comment by aslafy-z** [comment]
@r4zorgeek 

Make sure you are using at least node 10, check with `node -v` (you can use nvm: https://gist.github.com/d2s/372b5943bce17b964a79 to install these versions).

Then, start from the beginning:
```
git clone git@github.com:swapagarwal/swag-for-dev.git
cd swag-for-dev/
npm install
gulp
```

It works for us, it should work for you

**Comment by tsonge** [comment]
Hello. I decided to take a look at this since nobody responded in the past 2+ months. 

It seems like #374 had already fixed this issue, but was closed before merging. All that was needed was to sort the `swagList` in `get-data.js`, as it had done. If nobody's willing, perhaps I could open an identical PR, and see it through till it's merging?

That fixes most of the "jank" but there's still the "Choose tags..." drop-down menu that only appears after the window load event (thereby still leaving a bit of jank). Maybe that could be another issue?
